### PR TITLE
Adding details on changing log level for smaster and sminion

### LINKED ISSUE #489: SES7 Deployment Guide section on preparing for "ceph-salt apply" should mention "log_level: info"
**Summary**

The SES7 Deployment Guide definitely contains a chapter on deploying SES7 using ceph-salt and cephadm. This chapter must contain information on setting up Salt in preparation for running "ceph-salt apply".

This section should mention that:

1. the default log level of both salt-master and salt-minion is "WARNING"
2. the default log level is insufficient for debugging issues with Salt and ceph-salt
3. the user should change the log level setting to `log_level: info` for both salt-master and salt-minion
4. this setting should be changed on **all** cluster (minion) nodes

### PR DESCRIPTION
**Pull request by asettle**
Closes #489

### CODE REVIEW

File: xml/deploy_cephadm.xml [hunk 1]
```diff
@@ -158,6 +158,18 @@
      Install the <literal>salt-master</literal> on the &smaster; node:
     </para>
 <screen>&prompt.smaster;zypper in salt-master</screen>
+   </step>
+   <step>
+     <para>
+       Edit <filename>/etc/salt/master</filename> and uncomment the following
+       line:
+     </para>
+<screen>#log_level: WARN</screen>
```
Comment (smithfarm) [inline xml/deploy_cephadm.xml:167]: I'm thinking we should change `log_level` to `log_level_logfile` here - @ricardoasmarques Do you agree?

File: xml/deploy_cephadm.xml [hunk 2]
```diff
@@ -179,6 +191,24 @@
      Install the package <literal>salt-minion</literal> on all minion nodes.
     </para>
 <screen>&prompt.sminion;zypper in salt-minion</screen>
+   </step>
+   <step>
+     <para>
+       Edit <filename>/etc/salt/minion</filename> and uncomment the following
+       line:
+     </para>
+<screen>#log_level: WARN</screen>
```
Comment (smithfarm) [inline xml/deploy_cephadm.xml:200]: I'm thinking we should change `log_level` to `log_level_logfile` here

Comment (ricardoasmarques) [reply xml/deploy_cephadm.xml:200]: I think we only want to change `log_level_logfile`... but need to test it.

**Comment by asettle** [comment]
@smithfarm @ricardoasmarques - it would be good to close this one out. Any chance you've had to test? :) Thanks!

**Comment by ricardoasmarques** [comment]
@asettle I've not tested it and confirmed that `#log_level` must be replaced by `#log_level_logfile` on this PR.

**Review by ricardoasmarques — APPROVED** [review]
lgtm

**Pull request merged**